AutoAnswer/AutoAnswer/AppDelegate.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  AppDelegate.swift
+//  AutoAnswer
+//
+//  Created by Minhyuk Kim on 2021/06/11.
+//
+
+import UIKit
+import SendBirdCalls
+import Commons
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let appId: String = "94271307-58BC-45F1-9649-F078A2C1A08A"
```

Comment:
Remove and forcepush plz~ 🤣

Reply:
How do I miss this every time.. Fixed! :)